frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -258,9 +269,9 @@ const VariableDropdown: React.FC<VariableDropdownProps> = ({ id, name }) => {
   );
 };
 
-const AllVariableDropdowns = () => {
+const AllVariableDropdowns = ({ namespace }) => {
   const variables = useSelector(({ UI }: RootState) =>
-    UI.getIn(['monitoringDashboards', 'variables']),
+    UI.getIn(['monitoringDashboards', namespace ? 'dev' : 'admin', 'variables']),
```

Comment:
can we make use of `useActivePerspective()` here then concluding from namespace?